deployments/helm/KubeArmor/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
-appVersion: v1.0.3
+appVersion: v1.0.1
 description: A Helm chart for Kubearmor on Kubernetes
 home: https://github.com/kubearmor/KubeArmor
 icon: https://github.com/kubearmor/KubeArmor/blob/main/.gitbook/assets/logo.png?raw=true
 name: kubearmor
 type: application
-version: v1.0.3
+version: v1.0.1
```

Comment:
@DelusionalOptimist any reason for downgrading it here? AFAIK, we wanted this to be in sync with kubearmor releases.

Reply:
@kranurag7 we have been getting some bugs with the recent KubeArmor stable release (v1.0.3) we made. Thus we have reverted the `STABLE-RELEASE` file (which contains the stable release tag) to v1.0.1.
This PR is opened by one of the workflows to update the helm chart metadata files. Thus, the downgrade. Though it reads and replaces the `chartVersion` and `appVersion` using sed so we'll be pushing updates to v1.0.1 chart in the meantime anyways. This PR might not be needed.

Reply:
I'm thinking of changing the versioning of our charts, differentiating `appVersion` and `chartVersion` to prevent overwrites on previous chart version, which often creates a lot of confusion.

Reply:
Yes, I'll close this pull request. 

I am with you on having separate appVersion and chartVersion to avoid confusion :smile